command/debug/debug.go

```diff
@@ -133,6 +139,9 @@ func (c *cmd) init() {
 	c.help = flags.Usage(help, c.flags)
 
 	c.validateTiming = true
+	c.timeNow = func() time.Time {
+		return time.Now().UTC()
```

Comment:
I see we use both UTC and Local times. It looks like the usage is to use Localtime for output and UTC for files, but I've always found it a bit jarring to mix and match like that. My preference is UTC throughout, but I know reasonable people differ on this. Thoughts?

Reply:
I've made this change. For the time we print to stdout I used `time.Now().UTC()` and not this `c.timeNow`, because the intent of this `c.timeNow` is to make filenames predictable in tests, and if we call it extra times for stdout it becomes harder to predict the filenames.